app.js

```diff
@@ -1,59 +1,72 @@
 'use strict'
 
-let username = prompt('What is your name?').toUpperCase();
-alert('Hi' + username + 'lets play a game!');
+function username() {
+    let username = prompt('What is your name?').toUpperCase();
+    alert('Hi' + username + 'lets play a game!');
+}
+username();
```

Comment:
Try to call all your functions in one spot, the standard is at the end of your document. It's good to keep them all in one spot for easy debugging.